susemanager-utils/susemanager-sls/susemanager-sls.spec

```diff
@@ -84,15 +86,15 @@ mkdir -p %{buildroot}/usr/share/susemanager/formulas
 mkdir -p %{buildroot}/usr/share/susemanager/formulas/metadata
 mkdir -p %{buildroot}/usr/share/susemanager/reactor
 mkdir -p %{buildroot}/usr/share/susemanager/scap
-mkdir -p %{buildroot}/srv/formula_metadata
+mkdir -p %{buildroot}%{serverdir}/formula_metadata
```

Comment:
This might require also changes in java and in the was salt master is setup. I think formula data are read and written by java and other dirs which belong to this is configured in salt master config file.

Reply:
@mcalmer , this folder is empty on my prod instance. But I will be revisiting the Java code as well. There are lots of places that need updating.

For Enterprise Linux, I am happy to accept the risk for time being as a symlink would be a quick fix.

Reply:
Some background info: I think the idea of this directory is to provide a place where customer can add formuals. So it is equivalent to /etc/salt (for customer states) and /etc/pillar/ (for customer pillar data).
If you plan to move also these directories, which are part of the salt package, then this is correct.

Reply:
Thank you for that background. I already requested the move in https://github.com/openSUSE/salt-packaging/pull/54 and https://github.com/openSUSE/salt-packaging/pull/55

Reply:
This path is hardcoded in out external minion - https://github.com/uyuni-project/uyuni/blob/master/susemanager-utils/susemanager-sls/modules/pillar/suma_minion.py#L37

And also in documentation - https://github.com/uyuni-project/uyuni-docs/blob/master/modules/specialized-guides/pages/salt/salt-formulas-custom.adoc#file-structure-overview

That should all be changed to reflect this change.

Reply:
@aaannz , modified for both comments. (Doc update: https://github.com/uyuni-project/uyuni-docs/pull/1361) . Can you please review both for approach? `suma_minion.py` did my head in a bit on how to best handle it. Doc PR is my first contribution. Hopefully that approach is also ok.